src/en/discover/technology/index.html

```diff
@@ -164,15 +162,14 @@ <h2 class="h2 mb-12 lg:mb-16" id="file">File system</h2>
   <div class="grid lg:grid--cols-2 grid--gap-14 lg:grid--gap-28">
     <div>
       <p class="standout">
-        Traditional file system interface (CephFS) with POSIX compliance allowing for detailed metadata, dynamic rebalancing and high
-        performance within the same scalable system as object and block storage.
+      The Ceph File System (CephFS) provides a POSIX-compliant filesystem as a service that is layered on top of the object-based Ceph Storage Cluster. CephFS files are mapped to objects stored by Ceph in the Ceph Storage Cluster. Ceph clients mount CephFS filesystems as kernel objects or as Filesystems in User Space (FUSE).
```

Comment:
The Ceph File System (CephFS) is a robust, fully-featured POSIX-compliant distributed filesystem as a service with snapshots, quotas, and multi-cluster mirroring capabilities. CephFS files are striped across objects stored by Ceph for extreme scale and performance. Linux systems can mount CephFS filesystems natively, via a FUSE-based client, or via an NFSv4 gateway.